config/runtime.exs

```diff
@@ -11,6 +11,10 @@ config :skate,
   satellite_tileset_url: System.get_env("SATELLITE_TILESET_URL"),
   aws_place_index: System.get_env("AWS_PLACE_INDEX")
 
+config :skate, Skate.OpenRouteServiceAPI,
+  api_base_url: System.get_env("OPEN_ROUTE_SERVICES_API_URL"),
+  api_key: System.get_env("OPEN_ROUTE_SERVICES_API_KEY")
```

Comment:
__issue__: I don't think these should be plural?
```suggestion
  api_base_url: System.get_env("OPEN_ROUTE_SERVICE_API_URL"),
  api_key: System.get_env("OPEN_ROUTE_SERVICE_API_KEY")
```

Reply:
They should not be - you're right!